harvard-lts/drs-translation-service
# Console logging

**Adds console logging toggle**
* * *

**JIRA Issue**: https://at-harvard.atlassian.net/browse/LTSK8S-561

Testing:
1. Delete any logs in the logs/ folder for the project
2. Add CONSOLE_LOGGING_ONLY=true to the .env
3. Follow the [README](https://github.com/harvard-lts/drs-translation-service?tab=readme-ov-file#local-setup) instructions for local setup (NOTE: BB version is 2.4.2)
4. Spin it up locally
5. Verify that no logs appear in the logs/ directory
6. Bring down the container
7. Change CONSOLE_LOGGING_ONLY=false in the .env
8. Bring the container back up
9. Verify that logs appear in the logs/ directory

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no
